lib/components/Login/Login.tsx

```diff
@@ -44,7 +46,7 @@ export const Login = ({
           <ButtonGroup>
             <Button title="Login with Google" color="white" onClick={onClickHandler?.google} isFull />
             <Button title="Login with LinkedIn" color="white" onClick={onClickHandler?.linkedIn} isFull />
-            <Button title="Login with Twitter" color="white" onClick={onClickHandler?.twitter} isFull />
+            {useTwitter && <Button title="Login with Twitter" color="white" onClick={onClickHandler?.twitter} isFull />}
```

Comment:
Instead of adding a new prop we can handle it with something like below

`{onClickHandler?.twitter && (
              <Button title="Login with Twitter" color="white" onClick={onClickHandler?.twitter} isFull />
            )}`
            
            Apply same condition to all the social media button.

Reply:
@m-vojjala  Also add changelog please